coralproject/talk
# fix some config warnings

## What does this PR do?

fixing some config warnings so I can parse out other errors on the `suspect-wordlist` branch.

## How do I test this PR?

- load the Configure screen
- there should be no React warnings in the console.
- yay
